README.md

```diff
@@ -49,6 +49,7 @@ topCountry | no | '' | string | By providing this value you will tell component
 countryName | no | false | boolean | By setting this value to true, country names will be output in full instead of using the abbreviated short codes. Make sure to set this true for both country and region if you are using.
 whiteList | no | [] | array | Fill this array with capitalized short codes of the countries you want to appear in the dropdown list. ex: ['US', 'CA', 'MX']
 blackList | no | [] | array | Fill this array with capitalized short codes of the countries you want to remove from dropdown list. ex: ['US'] 
+class | no | [] | string | Class name ex: `form-control` 
```

Comment:
change class to className as class is a javascript keyword. Also change [] to '' as default value is string not array.